pkgs/tools/package-management/pdm/default.nix

```diff
@@ -79,14 +73,19 @@ buildPythonApplication rec {
   '';
 
   disabledTests = [
-    # sys.executable and expected executable are different
-    "test_set_non_exist_python_path"
+    # fails to locate setuptools (maybe upstream bug)
+    "test_convert_setup_py_project"
+    # requires network but is missing the mark
+    "test_actual_list_freeze"
+    "test_build_with_no_isolation"
+    "test_create_venv_first_time"
+    "test_create_venv_in_project"
+    "test_find_interpreters_from_venv"
+    "test_pep582_launcher_for_python_interpreter"
+    "test_run_with_another_project_root"
```

Comment:
Can you report these upstream, so they receive the network mark?

Reply:
I'm in the process of communicating with the maintainer and preparing a [PR](https://github.com/pdm-project/pdm/pull/1297) to address this

thus marking this PR as draft until this is fixed upstream

Reply:
solved